.drone.star

```diff
@@ -622,26 +622,39 @@ def website(ctx):
     },
     'steps': [
       {
-        'name': 'generate',
-        'image': 'webhippie/hugo:latest',
-        'pull': 'always',
+        'name': 'prepare',
+        'image': 'owncloud/alpine:latest',
         'commands': [
-          'make docs',
+          'apk add make',
+          'make docs-copy'
         ],
       },
       {
-        'name': 'publish',
-        'image': 'plugins/gh-pages:1',
-        'pull': 'always',
+        'name': 'test',
+        'image': 'klakegg/hugo:0.59.1-ext-alpine',
+        'commands': [
+          'cd hugo',
+          'hugo-official',
+        ],
+      },
+      {
+        'name': 'list',
+        'image': 'iankoulski/tree',
```

Comment:
Cool, now using some random guy docker image?

Reply:
better ideas?

Reply:
I'll include tree in the custom one I'll prepare

Reply:
owncloud/ubuntu got tree installed, I think owncloud/alpine as well